src/serialize.js

```diff
@@ -94,7 +94,7 @@ jQuery.fn.extend({
 			var type = this.type;
 
 			// Use .is( ":disabled" ) so that fieldset[disabled] works
```

Comment:
This comment needs updating to something like "Test by selector because the disabled _property_ ignores ancestor fieldsets".